modules/oe_theme_content_project/oe_theme_content_project.module

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file
+ * Module file used for theming the Project content type.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+
+/**
+ * Implements hook_ENTITY_TYPE_view_alter().
+ */
+function oe_theme_content_project_node_view_alter(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display) {
+  // Run only on project content type.
+  if ($entity->bundle() !== 'oe_project') {
+    return;
+  }
+
+  // Field labels are different in frontend and backend.
+  // Here we override them, per view mode.
+  if ($build['#view_mode'] === 'full') {
+    $label_overrides = [
+      'oe_project_dates' => t('Project duration'),
+      'oe_project_website' => t('Project website'),
+    ];
+    // If there is only a start date, set "Start date" as label.
+    $dates = $entity->get('oe_project_dates')->getValue();
+    if (!$dates[0]['end_value'] || $dates[0]['value'] === $dates[0]['end_value']) {
+      $label_overrides['oe_project_dates'] = t('Start date');
+    }
+
+    foreach ($label_overrides as $name => $label_override) {
+      if (!$entity->get($name)->isEmpty() && isset($build[$name]['#title'])) {
+        $build[$name]['#title'] = $label_override;
+      }
+    }
+  }
+}
+
+/**
+ * Implements hook_preprocess_field().
+ */
+function oe_theme_content_project_preprocess_field(&$variables) {
+  // Add ecl link classes to oe_project link fields.
+  if (in_array($variables['element']['#field_name'], ['oe_project_departments', 'oe_project_website'])) {
+    foreach ($variables['items'] as $key => $value) {
+      $variables['items'][$key]['content']['#options']['attributes']['class'][] = "ecl-link";
+      $variables['items'][$key]['content']['#options']['attributes']['class'][] = "ecl-link--standalone";
```

Comment:
Can we use here a Twig template? We can store them directly in the them /templatest/fields directory. It's always preferable, when possible, to use templates.

Reply:
The usage of template is now fixed.

Can you please explain me why not locate this templates in the `modules/oe_theme_content_project/templates/fields` ?
Thanks